lib/common/agents.py

```diff
@@ -1783,10 +1783,11 @@ def process_agent_packet(self, sessionID, responseName, taskID, data):
                 cur.execute("UPDATE results SET data=? WHERE id=?",(data, taskID))
```

Comment:
You made me realize there's another bug here where its not filtering on `agent` on the WHERE clause. You can fix it here or i'll put up another pull request for it. 😥

Reply:
I'll update that as well.